src/organisms/tables/ComparisonTable/index.js

```diff
@@ -1,35 +1,144 @@
 import React from 'react';
-import { Col, Layout } from 'atoms/Layout';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import {
+  Col,
+  Layout,
+} from 'atoms/Layout';
+import Spacer from 'atoms/Spacer';
+import Tooltip from 'atoms/Tooltip';
 
 import styles from './comparison_table.module.scss';
 
-const renderRow = (props, colClass = 'col') => {
+const renderRow = (props, className) => {
+  const {
+    children,
+    subHeader,
+    tooltip
+  } = props;
+
+  const addFloatingBorder = children.length === 3 && className !== 'col-header-3-cells' ? 'floating-border' : 'no-floating-border';
+
+  const classes = cx(
+    styles[className],
+    styles[addFloatingBorder],
+  );
+
+  const layoutProps = () => {
+    const layoutClassName = cx(styles.row, className === 'col-header-3-cells' && styles['header-3-row']);
+
+    if (children.length === 2) {
+      return {
+        className: layoutClassName,
+        smallCols: [ 12 ],
+        mediumCols: [ 4, 8 ]
+      };
+    }
+
+    return {
+      className: layoutClassName,
+      smallCols: [ 12, 6, 6 ],
+      mediumCols: [ 4 ]
+    };
+  };
+
+  const renderCol = (child, idx) => {
+    if (tooltip && idx === 0) {
+      return (
+        <Col className={classes}>
+          {child}
+          <Tooltip className={styles['tooltip']}>{subHeader}</Tooltip>
+        </Col>
+      );
+    }
+
+    if (subHeader && idx === 0) {
+      return (
+        <Col className={classes}>
+          {child}
+          <Spacer size={12} />
+          <Text
+            className={styles['tooltip']}
+          >
+            {subHeader}
+          </Text>
+        </Col>
+      );
+    }
+
+    return <Col className={classes}>{ child }</Col>;
+  };
+
+  return (
+    <Layout
+      {...layoutProps()}
+    >
+      { React.Children.toArray(children).map(renderCol) }
+
+      {
+        subHeader &&
+          <Col
+            className={styles['sub-header']}
+          >
+            <Text
+              size={10}
+              font='b'
+            >
+              {subHeader}
+            </Text>
+          </Col>
+      }
+    </Layout>
+  );
+};
+
+const renderHeaderRow = (props, className) => {
   const { children } = props;
-  const renderCol = child =>
-    <Col className={styles[colClass]} mediumCols={child.props.cols}>{child}</Col>;
+
+  const renderCol = child => <Col className={styles[className]}>{ child }</Col>;
 
   return (
-    <Layout className={styles['row']}>
+    <Layout
+      className={styles['header-row']}
+      smallCols={[ 12, 6, 6 ]}
+      mediumCols={[ 4 ]}
+    >
+      <Col />
       { React.Children.map(children, renderCol) }
     </Layout>
   );
 };
 
-const TableHeader = props => renderRow(props, 'col-head');
-const TableRow = props => renderRow(props);
+const TableHeader = props => props.children.length === 2 ? renderHeaderRow(props, 'col-header-2-cells') : renderRow(props, 'col-header-3-cells');
+const TableRow = props => renderRow(props, 'cell');
```

Comment:
Same thing with `renderRow`- you could probably just move `renderRow`'s function to `TableRow`, then in `TableHeader` call `TableRow` instead of `renderRow`?